broker/src/test/java/io/camunda/zeebe/broker/transport/backpressure/StabilizingAIMDLimitTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.broker.transport.backpressure;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.concurrent.TimeUnit;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class StabilizingAIMDLimitTest {
+
+  private StabilizingAIMDLimit limit;
+
+  @BeforeEach
+  void setup() {
+    limit =
```

Comment:
:wrench: As this is not dynamic, it could simply go in the declaration above.

```java
priva final StabilizingAIMDLimit limit =         StabilizingAIMDLimit.newBuilder()
            .initialLimit(10)
            .minLimit(1)
            .expectedRTT(1, TimeUnit.SECONDS)
            .build();
```